src/components/hmi_message_handler/src/messagebroker_adapter.cc

```diff
@@ -88,58 +88,85 @@ void MessageBrokerAdapter::processNotification(Json::Value& root) {
 
 void MessageBrokerAdapter::SubscribeTo() {
   LOG4CXX_INFO(logger_, "MessageBrokerAdapter::subscribeTo");
-  MessageBrokerController::subscribeTo("Buttons.OnButtonEvent");
-  MessageBrokerController::subscribeTo("Buttons.OnButtonPress");
-  MessageBrokerController::subscribeTo("UI.OnCommand");
-  MessageBrokerController::subscribeTo("VR.OnCommand");
-  MessageBrokerController::subscribeTo("BasicCommunication.OnReady");
+
+  // BasicCommunication
+  MessageBrokerController::subscribeTo("BasicCommunication.OnAppActivated");
```

Comment:
@AKalinich-Luxoft 
I believe we have to add changes for RC.* subscriptions only